ipq8064.c

```diff
@@ -130,99 +130,99 @@ static struct debug_mux cpul2_mux = {
+	{ .name = "sfpb_clk", .clk_mux = &ls_mux, .mux = 0x78 },
+	{ .name = "sfpb_a_clk", .clk_mux = &ls_mux, .mux = 0x78 },
+	{ .name = "pmic_ssbi2_clk", .clk_mux = &ls_mux, .mux = 0x7A },
+	{ .name = "pmic_arb0_p_clk", .clk_mux = &ls_mux, .mux = 0x7B },
+	{ .name = "pmic_arb1_p_clk", .clk_mux = &ls_mux, .mux = 0x7C },
+	{ .name = "prng_clk", .clk_mux = &ls_mux, .mux = 0x7D },
+	{ .name = "rpm_msg_ram_p_clk", .clk_mux = &ls_mux, .mux = 0x7F },
+	{ .name = "adm0_p_clk", .clk_mux = &ls_mux, .mux = 0x80 },
+	{ .name = "usb_hs1_p_clk", .clk_mux = &ls_mux, .mux = 0x84 },
+	{ .name = "usb_hs1_xcvr_clk", .clk_mux = &ls_mux, .mux = 0x85 },
+	{ .name = "usb_hsic_p_clk", .clk_mux = &ls_mux, .mux = 0x86 },
+	{ .name = "usb_hsic_system_clk", .clk_mux = &ls_mux, .mux = 0x87 },
+	{ .name = "usb_hsic_xcvr_fs_clk", .clk_mux = &ls_mux, .mux = 0x88 },
+	{ .name = "usb_fs1_p_clk", .clk_mux = &ls_mux, .mux = 0x89 },
+	{ .name = "usb_fs1_sys_clk", .clk_mux = &ls_mux, .mux = 0x8A },
+	{ .name = "usb_fs1_xcvr_clk", .clk_mux = &ls_mux, .mux = 0x8B },
+	{ .name = "tsif_p_clk", .clk_mux = &ls_mux, .mux = 0x8F },
+	{ .name = "tsif_ref_clk", .clk_mux = &ls_mux, .mux = 0x91 },
+	{ .name = "ce1_p_clk", .clk_mux = &ls_mux, .mux = 0x92 },
+	{ .name = "tssc_clk", .clk_mux = &ls_mux, .mux = 0x94 },
+	{ .name = "usb_hsic_hsio_cal_clk", .clk_mux = &ls_mux, .mux = 0x9D },
+	{ .name = "ce1_core_clk", .clk_mux = &ls_mux, .mux = 0xA4 },
+	{ .name = "pcie1_p_clk", .clk_mux = &ls_mux, .mux = 0xB0 },
+	{ .name = "pcie1_src_clk", .clk_mux = &ls_mux, .mux = 0xB1 },
+	{ .name = "pcie2_p_clk", .clk_mux = &ls_mux, .mux = 0xB2 },
+	{ .name = "pcie2_src_clk", .clk_mux = &ls_mux, .mux = 0xB3 },
+
+	{ .name = "afab_clk", .clk_mux = &hs_mux, .mux = 0x07 },
+	{ .name = "afab_a_clk", .clk_mux = &hs_mux, .mux = 0x07 },
+	{ .name = "sfab_clk", .clk_mux = &hs_mux, .mux = 0x18 },
+	{ .name = "sfab_a_clk", .clk_mux = &hs_mux, .mux = 0x18 },
+	{ .name = "adm0_clk", .clk_mux = &hs_mux, .mux = 0x2A },
+	{ .name = "sata_a_clk", .clk_mux = &hs_mux, .mux = 0x31 },
+	{ .name = "pcie_aux_clk", .clk_mux = &hs_mux, .mux = 0x2B },
+	{ .name = "pcie_phy_ref_clk", .clk_mux = &hs_mux, .mux = 0x2D },
+	{ .name = "pcie_a_clk", .clk_mux = &hs_mux, .mux = 0x32 },
+	{ .name = "ebi1_clk", .clk_mux = &hs_mux, .mux = 0x34 },
+	{ .name = "ebi1_a_clk", .clk_mux = &hs_mux, .mux = 0x34 },
+	{ .name = "usb_hsic_hsic_clk", .clk_mux = &hs_mux, .mux = 0x50 },
+	{ .name = "pcie1_aux_clk", .clk_mux = &hs_mux, .mux = 0x55 },
+	{ .name = "pcie1_phy_ref_clk", .clk_mux = &hs_mux, .mux = 0x56 },
+	{ .name = "pcie2_aux_clk", .clk_mux = &hs_mux, .mux = 0x57 },
+	{ .name = "pcie2_phy_ref_clk", .clk_mux = &hs_mux, .mux = 0x58 },
+	{ .name = "pcie1_a_clk", .clk_mux = &hs_mux, .mux = 0x66 },
+	{ .name = "pcie2_a_clk", .clk_mux = &hs_mux, .mux = 0x67 },
+
+	{ .name = "l2_m_clk", .clk_mux = &cpul2_mux, .mux = 0x2, 8 },
+	{ .name = "krait0_m_clk", .clk_mux = &cpul2_mux, .mux = 0x0, 8 },
+	{ .name = "krait1_m_clk", .clk_mux = &cpul2_mux, .mux = 0x1, 8 },
```

Comment:
think the last value is missing?

Reply:
It works out by total luck but you're right!